draft-ietf-ppm-dap.md

```diff
@@ -1270,14 +1274,14 @@ Before the Client can upload its report to the Leader, it must know the HPKE
 configuration of each Aggregator. See {{compliance}} for information on HPKE
 algorithm choices.
 
-Clients retrieve the HPKE configuration from each Aggregator by sending an HTTP
-GET request to `{aggregator}/hpke_config`.
+Clients retrieve the HPKE configuration from each Aggregator by sending a GET to
+`{aggregator}/hpke_config`.
 
-An Aggregator responds to well-formed requests with HTTP status code 200 OK and
-an `HpkeConfigList` value, with media type "application/dap-hpke-config-list".
-The `HpkeConfigList` structure contains one or more `HpkeConfig` structures in
-decreasing order of preference. This allows an Aggregator to support multiple
-HPKE configurations simultaneously.
+An Aggregator responds with an `HpkeConfigList`, with media type
+"application/dap-hpke-config-list". The `HpkeConfigList` contains one or more
+`HpkeConfig`s in decreasing order of preference. This allows an Aggregator to
+support multiple HPKE configurations and multiple sets of algorithms
+simultaneously.
```

Comment:
Here I remove the specific prescription to respond with 200 OK. I think this is consistent with [RFC 9205](https://datatracker.ietf.org/doc/html/rfc9205#section-3.1-1). I'd also point to [ACME](https://datatracker.ietf.org/doc/html/rfc8555#section-7.1.1), which describes what resource you get from some request, but only mentions status codes in non-normative examples.